hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/AbstractDatanodeStore.java

```diff
@@ -243,8 +253,7 @@ private static ColumnFamilyOptions buildColumnFamilyOptions(
         .setPinL0FilterAndIndexBlocksInCache(true)
```

Comment:
nit: I think we can remove lines 238-239 `RocksDB.loadLibrary` since this method is no longer static.

Reply:
> nit: I think we can remove lines 238-239 `RocksDB.loadLibrary` since this method is no longer static.

oh, by reading the comment in the code, it seems to be reasonable to remove this line, but I'll check and test it.

Reply:
@errose28 It seems that you are right, the `RocksDB.loadLibrary` is not needed anymore as I testd, will update, thx~